src/wallet/reconstruct_wallet_flow.c

```diff
@@ -261,14 +292,10 @@ reconstruct_state_e reconstruct_seed_handler(reconstruct_state_e state,
   return next_state;
 }
 
-/*****************************************************************************
- * GLOBAL FUNCTIONS
- *****************************************************************************/
-bool reconstruct_seed_flow(const uint8_t *wallet_id, uint8_t *seed_out) {
-  if ((NULL == wallet_id) || (NULL == seed_out)) {
-    return false;
-  }
-
+reconstruct_state_e reconstruct_wallet_secret_flow(
```

Comment:
static keyword is somehow missing in static functions present in these files. Can you please add

Reply:
updated, seems like I missed to comment here.